cmd/pj-rehearse/main_test.go

```diff
@@ -146,12 +147,19 @@ func getRehersalsHelper(logger *logrus.Entry, prNumber int) ([]*prowconfig.Presu
 	if err != nil {
 		return nil, fmt.Errorf("failed to load ci-operator config: %v", err)
 	}
+	ciopMasterConfig, err := config.CompoundLoad(masterCiopConfigPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load ci-operator config: %v", err)
+	}
 
 	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger)
 	if len(changedPresubmits) == 0 {
 		return nil, fmt.Errorf("Empty changedPresubmits was not expected")
 	}
 
+	changedCiopConfigs := diffs.GetChangedCiopConfigs(ciopMasterConfig, ciopPrConfig, logger)
+	changedPresubmits.AddAll(diffs.GetPresubmitsForCiopConfigs(prowPRConfig, changedCiopConfigs, logger))
```

Comment:
The fact that you needed to do this in the test, too, is smelly

Reply:
True, that test basically mimics our main().